src/components/Single-review.jsx

```diff
@@ -1,25 +1,33 @@
 import { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
-import { getSingleReview } from './api';
+import { getSingleReview, getCommentById } from './api';
 
 export const SingleReview = () => {
 	const [review, setReview] = useState([]);
+	const [comments, setComments] = useState([]);
 	let { review_id } = useParams();
 	const [isLoading, setIsLoading] = useState(true);
 	const [error, setError] = useState(null);
+	const [isComments, setIsComments] = useState(false);
+
+	const handleOnClick = () => {
+		setIsComments((isComments) => !isComments);
+	};
 
 	useEffect(() => {
 		const fetchData = async () => {
 			try {
 				const data = await getSingleReview(review_id);
+				const commentData = await getCommentById(review_id);
 				setReview(data);
+				setComments(commentData);
 			} catch (error) {
 				setError('Error fetching data from api');
 			}
 			setIsLoading(false);
 		};
 		fetchData();
-	}, [review_id]);
+	}, [review_id, isComments]);
```

Comment:
Adding isComments means that we're making these fetch requests every time someone collapses/opens the comment view. Is that what we want? 🤔 Could we have a way of keeping the data from the initial fetch request and only have it being shown/hidden instead?